docs/index.rst

```diff
@@ -23,17 +23,6 @@ API Reference
     datacatalog_v1beta1/types
 
 
-Migration Guide
```

Comment:
Owlbot keeps removing this section. I tried to add it back in but Owlbot just removed it. Possibly I need to make these changes elsewhere? cc: @tswast WWYD?

Reply:
I've pushed a commit to add `docs/UPGRADING.md` and the owlbot changes look good now. [Here](https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/common.py#L262) is the relevant synthtool code.